frontend/common/services/useTag.ts

```diff
@@ -68,8 +68,11 @@ export async function getTags(
   data: Req['getTags'],
   options?: Parameters<typeof tagService.endpoints.getTags.initiate>[1],
 ) {
-  store.dispatch(tagService.endpoints.getTags.initiate(data, options))
-  return Promise.all(store.dispatch(tagService.util.getRunningQueriesThunk()))
+  const result = await store.dispatch(
+    tagService.endpoints.getTags.initiate(data, options),
+  )
+  await Promise.all(store.dispatch(tagService.util.getRunningQueriesThunk()))
```

Comment:
I think here we can just adjust to 

```
return store.dispatch(
    tagService.endpoints.getTags.initiate(data, options),
  )
```
see the generated health code, I think this is hangover from an older version of the cli